src/components/balanceCard.jsx

```diff
@@ -27,7 +33,25 @@ const balanceCard = ({ token, isBank = true, hasBalance, isNativeToken }) => {
 
       setNeedsSync(isAccurateBalance);
     }
-  }, [token, isBank, daoMember, delegate]);
+
+    const getWnzAddress = async () => {
+      setWnzAddress(await getWrapNZap(daochain, daoid));
+    };
+    getWnzAddress();
+
+    if (wnzAddress) {
+      const rpcUrl = supportedChains[daochain].rpc_url;
+      const web3 = new Web3(new Web3.providers.HttpProvider(rpcUrl));
```

Comment:
We have a helper for getting web3 - the current provider is stored in the 'useInjectedProvider' context.

so you can import that and include the hook:

```
import { useInjectedProvider } from '../contexts/InjectedProviderContext';
...


const { injectedProvider } = useInjectedProvider();
...

injectedProvider.eth.getBalance()

```

Reply:
Thanks for the feedback! 
Changes committed and pushed.